pkg/cmd/server/api/types.go

```diff
@@ -474,6 +473,17 @@ type AuditConfig struct {
 	MaximumRetainedFiles int
 	// Maximum size in megabytes of the log file before it gets rotated. Defaults to 100MB.
 	MaximumFileSizeMegabytes int
+
+	// Path to the file that defines the audit policy configuration.
+	PolicyFile string
```

Comment:
If this isn't specified, the default behavior out should remain consistent with what it was before, right?  Log everything as I recall?  Either way, it should be described.

Reply:
If this file isn't specified, we fallback to the old audit scheme. So that after an upgrade we work the same way, as we did before. I turn the advanced audit on this config value being present.

Reply:
> If this file isn't specified, we fallback to the old audit scheme. So that after an upgrade we work the same way, as we did before. I turn the advanced audit on this config value being present.

You need to document what it does.  It seems weird that not specifying a file and specifying an empty file give two different outcomes.

Reply:
An empty file means you wanted to give it and you just messed up its contents. Its lack, though means we're working in the old audit mode, for backwards compatibility. I'll be updating our docs with the advanced bits.